lib/common/strings.c

```diff
@@ -1121,13 +1121,11 @@ pcmk__strcmp(const char *s1, const char *s2, uint32_t flags)
             crm_err("Bad regex '%s' for update: %s", s2, strerror(regcomp_rc));
         } else {
             rc = regexec(&r_patt, s1, 0, NULL, 0);
-
+            regfree(&r_patt);
```

Comment:
regcomp(3p) (i.e. posix): "Upon successful completion, the regcomp() function shall return 0. Otherwise, it shall return an integer value indicating an error as described in \<regex.h\>, and the content of preg is undefined." and "If the preg argument to regexec() or regfree() is not a compiled regular expression returned by regcomp(), the result is undefined."

Reply:
Oh okay, the (3) man page is installed on my system but the (3p) one isn't.